### DISCUSSION TITLE
Mod idea: Kilrathi ship

### DISCUSSION THREAD

**Discussion by DominusExult** [exult/exult#318] [General]
a mod idea for the Bloodfang-class Superfighter you find on Mack of Britain's field:

- when you click on it the ship's computer speaks to you in an unrecognizable language
- you ask Mack about it and he mentions that he kept the head of the "tigerlion" he ate.
- either have it already be prepared or add another little side quest of finding someone who can prepare the head as some kind of helmet
- once you wear that you can understand the computer and that it says something about only having three seats
- get rid of excess party members and try again
- some special effect that makes your party disappear and the ship takes off
- on a new map you find yourself inside the ship (new barge) in space next to the planet Britannia
- some kind of ships hold with Kilrathi helmets and space suits and an energy weapon
- energy weapon uses batteries and shoots beams (paralyze shape 408?) doing a lot of damage (maybe limit its use to outside of Britannia)
- a computer device that once you doubleclick makes you sit down and the ship travels to some planet, a Kilrathi outpost or whatever (maybe have the computer ask for several possible locations but only allow one and the others are out of reach because of fuel/energy being low)
- you click on the planet and again some special effect landing you on this planet (maybe some island surrounded with water) and placing the party outside of the Bloodfang ship
- outside you meet Kilrathi (using SI's Yuriel as a base shape?)
- They will only speak to you if you wear the helmets/spacesuits. If you don't or wear the tigerlion's head they will attack you.
- some kind of quest or just time to explore the outpost.

As a reward you get @marzojr "portable hole" (limitless container) <- so maybe implement it with the Keyring mod.

Things you need to create for this mod:
- the Bloodfang barge in SciFi look (seats, container, computer device)
- graphics for tigerlion head, helmets, spacesuits, energy weapon, batteries for the weapon
- Kilrathi NPCs
- Outpost building graphics in SciFi look
- an idea on how to implement the take off/landing of the Bloodfang

**Comment by DominusExult** [comment]
To tie up a loose Wing Commander end, this Bloodfang could have an experimental time warp engine that brings you into the future. The planet you land is actually an abandoned outpost of the Nephilim (Wing Commander Prophecy enemy) and you find imprisoned Kilrathi and Commander Blair (or just the dead body and a diary?).
Or to make it easier, Blair and the Kilrathi have escaped from the Nephilim and crash landed there and had to build huts (so no SciFi buildings needed).
Possible story line:
- they need to fix their ship but a crucial part (energy core) was taken by monsters into caves. Only you with the energy weapons can retrieve it
- some part of the cave is closed off with blackrock (so you need to have Rudyom's wand)
- once you retrieve the core you need to transfer energy from your ship, essentially leaving with just enough energy to make it home to Britannia
- possible to have Blair in your party to retrieve the core (but a check that prevents him from entering the Bloodfang ship)
- witty conversation about earth and that they are from a different time (must mention the sports almanac as in Back to the Future)

**Comment by ShadowTigerZC** [comment]
I would give my left kidney to see this quest in-game.

I'm still amazed how small the Kilrathi ship is in person relative to how big it is canonically.  The thing is barely as wide as a dozen stalks of wheat in-game.  


![KilrathiStrakha2Ultimra7](https://user-images.githubusercontent.com/10102562/226039902-862a834e-6137-45e8-9aa5-ab01a330eb42.png)
![KilrathiStrakha1](https://user-images.githubusercontent.com/10102562/226039907-fbd142d2-59e4-4d6e-9bff-4ac995f5b52a.png)
